python/simple_portfolio_data.py

```diff
@@ -2,10 +2,11 @@
 
 np.random.seed(1)
 n = 20
-pbar = np.ones((n,1))*.03 + np.r_[np.random.rand(n-1,1), np.zeros((1,1))]*.12;
-S = np.random.randn(n, n); S = np.asmatrix(S)
+pbar = np.ones((n,1))*.03 + np.r_[np.random.rand(n-1,1), np.zeros((1,1))]*.12
+S = np.random.randn(n, n)
 S = S.T*S
 S = S/max(np.abs(np.diag(S)))*.2
-S[:, -1] = np.zeros((n, 1))
-S[-1, :] = np.zeros((n, 1)).T
-x_unif = np.ones((n, 1))/n; x_unit = np.asmatrix(x_unif)
+S[:, -1] = np.zeros(n)
+S[-1, :] = np.zeros(n).T
+x_unif = np.ones((n, 1))/n
```

Comment:
Any reason to make x_unif a (n, 1) instead of (n,) shaped array?

Reply:
Nope. Updated.